rootdir/etc/fstab.mt6755

```diff
@@ -7,7 +7,9 @@
 /dev/block/platform/mtk-msdc.0/11230000.msdc0/by-name/boot         /boot               emmc       defaults                                                      defaults
 /dev/block/platform/mtk-msdc.0/11230000.msdc0/by-name/system       /system             ext4       ro                                                            wait
 /dev/block/platform/mtk-msdc.0/11230000.msdc0/by-name/userdata     /data               ext4       noatime,nosuid,nodev,noauto_da_alloc,discard                  wait,check,resize,encryptable=/dev/block/platform/mtk-msdc.0/11230000.msdc0/by-name/metadata,
+/dev/block/platform/mtk-msdc.0/11230000.msdc0/by-name/userdata     /data               f2fs       noatime,nosuid,nodev,noauto_da_alloc,discard                  wait,check,resize,encryptable=/dev/block/platform/mtk-msdc.0/11230000.msdc0/by-name/metadata,
 /dev/block/platform/mtk-msdc.0/11230000.msdc0/by-name/cache        /cache              ext4       noatime,nosuid,nodev,noauto_da_alloc,discard                  wait,check
+/dev/block/platform/mtk-msdc.0/11230000.msdc0/by-name/cache        /cache              f2fs       noatime,nosuid,nodev,noauto_da_alloc,discard                  wait,check
```

Comment:
for cache the noatime,nosuid,nodev,noauto_da_alloc,discard                  wait,check
and for data noatime,nosuid,nodev,noauto_da_alloc,discard                  wait,check, resize,encryptable=/dev/block/platform/mtk-msdc.0/11230000.msdc0/by-name/metadata,

also the ( , ) after the metadata, is it correct?